e2e/fixtures/orders.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [
+  {
+    orderTitle: 'Example Order Title',
+    orderDetails: 'Example order details here - Lorem ipsum dolor sit amet...',
+    orderDoc: 'C21_order.pdf',
+    judgeAndLegalAdvisor: {
```

Comment:
It's perhaps worth removing this from the orders object and creating a new judge fixture.

Reply:
Good idea, but I'd like to do that change maybe when we have time to update all the e2e tests that currently input a judge so that they all use a judge fixture. So maybe not right now

Reply:
Yeah agreed we can do as tech debt.